protocols/CopyToResultsDir.sh

```diff
@@ -59,22 +59,22 @@ cp "${projectJobsDir}/${project}.csv" "${projectResultsDir}"
 usedWorkflow=$(basename ${workflow})
 
 	cp "${intermediateDir}"/*.sorted.merged.bam "${projectResultsDir}/alignment"
-        cp "${intermediateDir}"/*.sorted.merged.bam.md5 "${projectResultsDir}/alignment"
-        cp "${intermediateDir}"/*.sorted.merged.bai "${projectResultsDir}/alignment"
-        cp "${intermediateDir}"/*.sorted.merged.bai.md5 "${projectResultsDir}/alignment"
+        cp "${intermediateDir}"/*.sorted.merged.bam.md5sum "${projectResultsDir}/alignment"
```

Comment:
what you can do is this:
`cp "${intermediateDir}"/*.sorted.merged.bam.{md5sum,bai,bai.md5sum} "${projectResultsDir}/alignment"`
Then you will cover also lines 63 and 64